### ISSUE TITLE
Minimum Android SDK is API 21? 

### ISSUE THREAD

**Issue by okuhlen** [VirgilSecurity/virgil-sdk-java-android#64]
Hi,

Just wanted to get clarification as to what the minimum SDK is for this sdk... recently discovered VirgilSecurity and would love to add this to my android app project. According to the repo's introductionary page, the minimum API level is 16+. When I try include the latest version of this project, that being:     implementation "com.virgilsecurity.sdk:sdk-android:7.1.0", I get an error, suggesting I should update my minSdkVersion to 21: 

`Manifest merger failed : uses-sdk:minSdkVersion 19 cannot be smaller than version 21 declared in library [com.virgilsecurity.sdk:sdk-android:7.1.0]`

Is API 21 the minimum for this project, or is there something else I should do to get it working on KitKat? 

Thanks.

**Comment by SergeySeroshtan** [comment]
Yes, minimum API version is 21. We do not plan to support lower versions.
